fabrictestbed_extensions/fablib/network_service.py

```diff
@@ -349,10 +350,17 @@ def new_network_service(
         fim_network_service = slice.topology.add_network_service(
             name=name, nstype=nstype, interfaces=fim_interfaces
         )
-
+        logging.debug(fim_network_service)
         network_service = NetworkService(
             slice=slice, fim_network_service=fim_network_service
         )
+        # Check service type for external connectivity:
+        if isinstance(nstype, type(ServiceType.FABNetv4Ext)):
+            user_data["service type"] = "FABNetv4Ext"
```

Comment:
Just wanted to  check if the space in the key name intentional. Ignore my comment if it is.

Reply:
It does not need to have a space. Let me know if I should change this. I am appending a new key value pair to user_data here (not changing an existing one).

Reply:
I would suggest using `service_type` instead of space.

Reply:
Updated!